tools/src/scie_pants/pants_python_versions.json

```diff
@@ -0,0 +1,4 @@
+[
+  {"pants": "2.5.0.dev0", "python": "cp39"},
+  {"pants": "2.0.0.dev0", "python": "cp38"}
+]
```

Comment:
Yea, add to *the top* of the list, and it would be `cp312` and we also need to add that to the list of cpython interpreters in the lift manifest and known python versions.

I'll add a section in the readme about how to add a new python to the list of supported pythons along with how to declare a new cpython version break for Pants.

Reply:
inlined in the python code, with a comment.